nix/dhall-docs-gen.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, dhall-docs, unzip, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "dhall-docs-artifacts";
+
+  src = fetchurl {
+    url = "https://github.com/dhall-lang/dhall-lang/archive/v17.0.0.zip";
+    sha256 = "8c46f81d6131bf5dac715a3f107d0a43f4e8decaadd7ed36fbc4375b5b32eeaf";
+  };
+
+  buildInputs = [ dhall-docs unzip ];
+
+  installPhase = ''
+    mkdir -p $out/nix-support/hydra-build-products
+    dhall-docs --input ./Prelude --output $out/nix-support/hydra-build-products/Prelude > $out/nix-support/hydra-build-products/dhall-docs.log
```

Comment:
I believe if you add this line then Hydra will server the generated HTML and the log from its user interface:

```
echo "report html $out/nix-support/hydra-build-products/Prelude/index.html" >> "$out/nix-support/hydra-build-products"
echo "report log $out/nix-support/hydra-build-products/dhall-docs.log" >> "$out/nix-support/hydra-build-products"
```

Reply:
I had to `report html` the folder where docs are generated, and it works! If you go here https://hydra.dhall-lang.org/build/62246 you can see the html report link, which is: https://hydra.dhall-lang.org/build/62246/download/1/docs/

I think that suffices the task. Currently the prelude at master doesn't have any .dhall files, so little to no files are actually generated.